tests/FakeItEasy.Tests/Core/MethodInfoManagerTests.cs

```diff
@@ -211,7 +211,7 @@ public void DoSomething(string text)
             }
 
             [SuppressMessage("Microsoft.Usage", "CA1801:ReviewUnusedParameters", MessageId = "other", Justification = "Required for testing.")]
-            public bool Equals(Base other) => true;
+            public bool Equals([AllowNull] Base other) => true;

```

Comment:
The interface method is declared with this attribute, not having it on the implementation caused a warning.